pkgs/development/libraries/mlpack/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, armadillo, boost, cmake, fetchFromGitHub, ensmallen
+, enableOpenMP ? false }:
+
+stdenv.mkDerivation rec {
+  name = "mlpack-${version}";
+  version = "3.2.1";
+
+  src = fetchFromGitHub {
+    owner = "mlpack";
+    repo = "mlpack";
+    rev = version;
+    sha256 = "sha256-/UN/Pmt7m3s9pq4I3XTt0AbHEj3nbbdWeLLcs1TE+9w=";
+  };
+
+  buildInputs = [ cmake armadillo boost ensmallen ];
```

Comment:
Please put dependencies that are **not** needed at run-time in `nativeBuildInputs`.
It works this way too, but it will break if somebody tries to cross-compile it.
In this case and in the other package, just `cmake`

Reply:
oh makes sense